python/pp_ser.py

```diff
@@ -659,6 +638,7 @@ def __re_endmodule(self):
             if self.__module != m.group(2):
                 self.__exit_error(msg = 'Was expecting "end '+m.group(1)+' '+self.__module+'"')
             self.__module = ''
+            self.__useStmtInModule = False
```

Comment:
@pspoerri It is placed at line 574 (see my comment on this line). Here, we reached the `END MODULE` statement so we reset the variable in case the translation unit contains another `MODULE` or standalone `SUBROUTINE` / `FUNCTION`